src-ui/lib/drizzle/relations.ts

```diff
@@ -1,3 +1 @@
-import { relations } from "drizzle-orm/relations";
-import {  } from "./schema";
-
+import { } from './schema'
```

Comment:
likely not a necessary change, mostly satisfying eslint -- but I've since ignored this folder in eslint. 🤷 gonna keep the change for now though.